src/library/basesqltablemodel.cpp

```diff
@@ -32,10 +34,10 @@ BaseSqlTableModel::BaseSqlTableModel(QObject* pParent,
           m_database(pTrackCollection->getDatabase()),
           m_previewDeckGroup(PlayerManager::groupForPreviewDeck(0)),
           m_iPreviewDeckTrackId(-1),
-          m_currentSearch("") {
-    m_bInitialized = false;
-    m_iSortColumn = 0;
-    m_eSortOrder = Qt::AscendingOrder;
+          m_bInitialized(false),
+          m_currentSearch(""),
+          m_trackSourceSortColumn(kIdColumn),
```

Comment:
(begin pedant mode!)

A track source column is not the same as a table column. See ```BaseSqlTableModel::fieldIndex```. So using the constant for the ID table column is not guaranteed to be the position of the ID column in the track source (it is in practice). I'd leave it as 0 so you're saying by default sort by the first track source column, whatever that is.